apps/pwabuilder/src/script/components/windows-form.ts

```diff
@@ -422,6 +504,19 @@ export class WindowsForm extends AppPackageFormBase {
                     (this.packageOptions.images!.baseImage = val),
                 })}
               </div>
+              <div class="form-group">
+                ${this.renderColorPicker({
```

Comment:
This color picker logic also seems like a good candidate to split out into another file

Reply:
I'll move this to `app-package-form-base.ts` since the android form also uses color pickers in another PR